pkg/migration/runner.go

```diff
@@ -219,6 +222,16 @@ func (r *Runner) Run(originalCtx context.Context) error {
 	}
 	r.logger.Info("copy rows complete")
 
+	// r.handleDeferredCutOver may return an error if there is
+	// some unexpected problem checking for the existence of
+	// the sentinel table OR if sentinelWaitLimit is exceeded.
+	// This function is invoked even if DeferCutOver is false
+	// because it's possible that the sentinel table was created
+	// manually after the migration started.
+	if err := r.handleDeferredCutOver(ctx); err != nil {
+		return err
+	}
```

Comment:
I prefer the naming "waitOnSentinelTable" or something of that nature so it's clear it's a block wait.